client/lib/analytics/index.js

```diff
@@ -512,5 +507,30 @@ const analytics = {
 		window._tkq.push( [ 'clearIdentity' ] );
 	},
 };
+
+/**
+ * Wrap Google Analytics with debugging, possible analytics supression, and initialization
+ *
+ * This method will display debug output if Google Analytics is suppresed, otherwise it will
+ * initialize and call the Google Analytics function it is passed.
+ *
+ * @see isGoogleAnalyticsAllowed)
+ *
+ * @param  {Function} func Google Analytics tracking function
+ * @return {Function}      Wrapped function
+ */
+function makeSafeDebuggableGoogleAnalyticsFunc( func ) {
```

Comment:
Sorry, I really don't know what to call this…

Reply:
It's not too bad. Maybe ditch `safe`?

Reply:
Yeah, it initialized and protects agains disallowed calls. Maybe even just `makeGoogleAnalyticsTrackingFunction`?

Reply:
Sounds better, go for it 👍

Reply:
`makeAnalyticsGreatAgain`